knowledge-hub/labels.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "Label Management"
+description: "Creating, managing and tuning labels for feedback triage."
+---
+
+## Overview
+
+Project area labels and tag labels are workspace-defined labels that help triage feedback snippets.
+When auto-triage is enabled for a particular label, Monterey will go ahead and auto assign a project area label when the feedback is ingested or the labels are backfilled, provided our AI models are confident in that prediction.
+
+While project area labels and tag labels work functionally similar in downstream uses (filtering, chat, reports, etc.), they have one key difference that necessitates the separation of the two.
+
+For a particular snippet when auto-triage is enabled:
+
+- 1 or 0 project areas can be auto-assigned
+- Many or 0 tag areas can be auto-assigned
+
+Thus, it is often the case that there are many more tags applied since one snippet may have multiple.
+We suggest keeping this distinction in mind when building out labels for each of these.
+
+_Note: If a feedback item is split into multiple snippets, where each snippet has a different project area, you will see the parent feedback with multiple project areas.
+Additionally, you can also always manually add more than one project area label._
+
+--
+
+Auto-triage performance will be influenced by a combination of clear label names, detailed label descriptions, and human-labeled examples that are captured throughout the workspace.
```

Comment:
I think this line then begs the question -- how should descriptions be written. Prob a good opportunity to include some content somewhere of like "here's a few examples of descriptions that work well in the system" and even maybe, "here's a few examples that don't help" (e.g. where the description is something like `feedback about {label name}` lol)

Reply:
true true added a little more color there